src/main/kotlin/com/openlattice/linking/controllers/RealtimeLinkingController.kt

```diff
@@ -52,6 +51,23 @@ class RealtimeLinkingController(
         return linkableEntitySets.minus(entitySetsNeedLinking)
     }
 
+    @RequestMapping(
+            path = [RealtimeLinkingApi.MISSING],
+            method = [RequestMethod.POST],
+            produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun getEntitiesMissingLinking(@RequestBody entitySetIds: Set<UUID>): Map<UUID, Set<UUID>> {
+        ensureAdminAccess()
+        val linkableEntitySets = lqs
+                .getLinkableEntitySets(linkableTypes, entitySetBlacklist, whitelist.orElse(setOf()))
+                .toSet()
+        val entitiesNeedLinking = HashMap<UUID, Set<UUID>>()
+                linkableEntitySets.forEach {
```

Comment:
This should query the requested entity set ids and not all linkable entity set ids